### DISCUSSION TITLE
Tech Stack

### DISCUSSION THREAD

**Discussion by mnichols08** [chingu-voyages/v48-tier1-team-04#32] [Polls]
We must decide how we want to write our application.  Should we leverage a library?

**Comment by sebastiandlq** [comment]
Well kinda naive, but Mike's article really made me want to try the model view control model

**Reply by JOSIK95** [reply]
I think I would like to try the MVC but I am open to anything

**Comment by vinccodes** [comment]
I am open to MVC, React.js I am familiar with the basics of useState(), useEffect() and passing props into individual components. I am also not opposed to using Node.js with Express backend and EJS templating engine to build a traditional web application that is not a SPA.

**Comment by mnichols08** [comment]
We have 3/5 votes. But 2/3 say lets build it line-by-line. I am cool with this.

Keep in mind, we can always opt to refactor the code at any point and should still try to write efficient code and often refactor it to improve anyway.
